### ISSUE TITLE
is there a pytorch version?

### ISSUE THREAD

**Issue by tb1over** [datapplab/AutoClass#1]
hello, is there a pytorch version for autocalss ? thx

**Comment by wluo1** [comment]
No, we only have the tensorflow version. Users are more than welcome to implement AutoClass in pytorch if they are interested. Thanks.
